src/Subscription.ts

```diff
@@ -13,7 +13,7 @@ export type TeardownLogic = AnonymousSubscription | Function | void;
 
 export interface ISubscription extends AnonymousSubscription {
   unsubscribe(): void;
-  closed: boolean;
+  readonly closed: boolean;
```

Comment:
👍 this will encourage TypeScript users to not try to modify this value (as it shouldn't be mutable)